Atomic/util.py

```diff
@@ -119,7 +123,7 @@ def default_ro_container_context():
 
 def write_out(output, lf="\n"):
     sys.stdout.flush()
-    sys.stdout.write(str(output) + lf)
+    sys.stdout.write(output + lf)
```

Comment:
For example it looks like the `str` there came from 6e1f4dde ...which seems like it was probably copied from somewhere for a reason.  I'm guessing it's something like needing to handle stdout being passed unicode but expecting bytes?